src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -15,7 +15,7 @@ export default function DocumentList({ documents }) {
       <ul>
         {documents &&
           documents.map((document) => (
-            <li key={document.name}>
+            <li key={document.identifier}>
```

Comment:
We have 2 areas (PackageReview and SubmissionReview) where we display a document list and they don't have the exact same look and feel. I wonder if we shouldn't have a component in this app that could display a list of documents.

Reply:
![image](https://user-images.githubusercontent.com/51387119/106772301-b6e7c400-65f4-11eb-98fe-b7d9a191dd3a.png)
that the list of documents from PackageReview

Reply:
They are not exactly the same but still, they are close and built totally differently

Reply:
It sounds good in theory, but these lists are quite different.  The SubmissionReview is about displaying the fees and description of the uploaded docs.  The PackageReview is about displaying the status of the file with links to withdraw or re-submit.  

Also, this is how the SubmissionReview looks on mobile, I think it's a bug: 
![image](https://user-images.githubusercontent.com/55215368/106781541-07175400-65fe-11eb-9886-d9e31c1b7679.png)

Reply:
agree @KDNeufeld , let's discuss it

Reply:
decision to review with follow-up ticket, FLA-752